js/MineSweeper.js

```diff
@@ -215,19 +217,28 @@ jQuery(function ($) {
                 } else {
                     input.prop('disabled', true);
                 }
-                $('.new-game').trigger('click');
+                $('#newGame').trigger('click');
```

Comment:
please change back to using a css class instead of id, it makes everything more versatile and portable.